static/app/views/dashboardsV2/manage/index.tsx

```diff
@@ -324,6 +324,10 @@ class ManageDashboards extends AsyncView<Props, State> {
   }
 }
 
+const StyledTitle = styled(Title)`
```

Comment:
Is there actually something we can change in `StyledPageHeader` to fix this so we don't have to introduce a 2nd styled component?

Reply:
Not that I could find, but honestly I'm not sure why this is happening in the first place in Safari, landed on this fix by guessing and checking.